apm-sniffer/apm-toolkit-activation/apm-toolkit-trace-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/trace/ActiveSpanActivation.java

```diff
@@ -35,11 +35,13 @@
  *
  * @author zhangxin
  */
-public class ActiveSpanTagActivation extends ClassStaticMethodsEnhancePluginDefine {
+public class ActiveSpanActivation extends ClassStaticMethodsEnhancePluginDefine {
```

Comment:
Why change this name?

Reply:
because  ActiveSpan contains only one tag before, but now ActiveSpan contains two method .so i think ActiveSpanActivation is better, what's ur idea?

Reply:
I see. Make sense.